network/p2p/utils_test.go

```diff
@@ -0,0 +1,40 @@
+package p2p
```

Comment:
I think we called it test_utils up until now so we should be consistent

Reply:
agree, will change it